### PR TITLE
[v1.4] Adam's PPP

### PR DESCRIPTION
**Pull request by adamwth**
=== User Guide
- Update Terminology section
- Add icons to clearly display important info
- Add Reassign command

=== Developer Guide
- Add activity diagram for check-in/out

=== Portfolio
- Added my PPP

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ ifdef::env-github[]
 endif::[]
 :repoURL: https://github.com/CS2103-AY1819S1-F11-2/main
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Team Concierge(TM)`      Since: `Jun 2016`      Licence: `MIT`
```
Comment (pikulet) [inline docs/DeveloperGuide.adoc]: I'm actually not sure if we have to credit the other team. Anyway, should be no issue with this.

The date can be changed to Aug 2018 too.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -244,53 +244,53 @@ This section describes some noteworthy details on how certain features are imple
...
 
 image::UndoRedoNewCommand2StateListDiagram.png[width="800"]
 
 [NOTE]
-If a command fails its execution, it will not call `Model#commitConcierge()`, so Concierge state will not be saved into the `conciergeStateList`.
+If a command fails its execution, it will not call `Model#commitConcierge()`, so Concierge(TM) state will not be saved into the `conciergeStateList`.
 
-Step 4. The user now decides that adding the guest was a mistake, and decides to undo that action by executing the `undo` command. The `undo` command will call `Model#undoConcierge()`, which will shift the `currentStatePointer` once to the left, pointing it to the previous Concierge state, and restores Concierge to that state.
+Step 4. The user now decides that adding the guest was a mistake, and decides to undo that action by executing the `undo` command. The `undo` command will call `Model#undoConcierge()`, which will shift the `currentStatePointer` once to the left, pointing it to the previous Concierge(TM) state, and restores Concierge(TM) to that state.
```
Comment (pikulet) [inline docs/DeveloperGuide.adoc]: Okay at this point the Concierge(TM) is a bit weird...

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -208,12 +208,12 @@ image::ModelClassDiagram.png[width="800"]
...
 
 * stores a `UserPref` object that represents the user's preferences.
-* stores Concierge data.
+* stores Concierge(TM) data.
 * exposes an unmodifiable `ObservableList<Guest>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
 * does not depend on any of the other three components.
 
 [NOTE]
-As a more OOP model, we can store a `Tag` list in `Concierge`, which `Guest` can reference. This would allow `Concierge` to only require one `Tag` object per unique `Tag`, instead of each `Guest` needing their own `Tag` object. An example of how such a model may look like is given below. +
+As a more OOP model, we can store a `Tag` list in `Concierge(TM)`, which `Guest` can reference. This would allow `Concierge(TM)` to only require one `Tag` object per unique `Tag`, instead of each `Guest` needing their own `Tag` object. An example of how such a model may look like is given below. +
```
Comment (pikulet) [inline docs/DeveloperGuide.adoc]: This `Concierge` is referring to the class I believe.

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -997,29 +1004,29 @@ Do take a look at <<Design-Logic>> before attempting to modify the `Logic` compo
...
 [TIP]
 Do take a look at <<Design-Model>> before attempting to modify the `Model` component.
 
-. Add a `removeTag(Tag)` method. The specified tag will be removed from everyone in Concierge.
+. Add a `removeTag(Tag)` method. The specified tag will be removed from everyone in Concierge(TM).
 +
 ****
 * Hints
-** The link:{repoURL}/src/main/java/seedu/address/model/Model.java[`Model`] and the link:{repoURL}/src/main/java/seedu/address/model/Concierge.java[`Concierge`] API need to be updated.
+** The link:{repoURL}/src/main/java/seedu/address/model/Model.java[`Model`] and the link:{repoURL}/src/main/java/seedu/address/model/Concierge(TM).java[`Concierge(TM)`] API need to be updated.
```
Comment (pikulet) [inline docs/DeveloperGuide.adoc]: Concierge is probably referring to the class

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -997,29 +1004,29 @@ Do take a look at <<Design-Logic>> before attempting to modify the `Logic` compo
...
 ** Think about how you can use SLAP to design the method. Where should we place the main logic of deleting tags?
-**  Find out which of the existing API methods in  link:{repoURL}/src/main/java/seedu/address/model/Concierge.java[`Concierge`] and link:{repoURL}/src/main/java/seedu/address/model/guest/Guest.java[`Guest`] classes can be used to implement the tag removal logic. link:{repoURL}/src/main/java/seedu/address/model/Concierge.java[`Concierge`] allows you to update a guest, and link:{repoURL}/src/main/java/seedu/address/model/guest/Guest.java[`Guest`] allows you to update the tags.
+**  Find out which of the existing API methods in  link:{repoURL}/src/main/java/seedu/address/model/Concierge(TM).java[`Concierge(TM)`] and link:{repoURL}/src/main/java/seedu/address/model/guest/Guest.java[`Guest`] classes can be used to implement the tag removal logic. link:{repoURL}/src/main/java/seedu/address/model/Concierge(TM).java[`Concierge(TM)`] allows you to update a guest, and link:{repoURL}/src/main/java/seedu/address/model/guest/Guest.java[`Guest`] allows you to update the tags.
```
Comment (pikulet) [inline docs/DeveloperGuide.adoc]: Concierge is probably referring to the class.

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (adamwth) [reply docs/DeveloperGuide.adoc]: We credited in the README already, so i think should be fine? 😃 

updated in latest commit

**Comment by pikulet** [comment]
👍 can try to fix the images in a future PR

**Pull request merged**